### DISCUSSION TITLE
Digital/lab8/tasks/

### DISCUSSION THREAD

**Discussion by giscus** [wintermelon008/USTC-DigitalLab-Discussion#23] [Announcements]
# Digital/lab8/tasks/

https://soc.ustc.edu.cn/Digital/lab8/tasks/

**Comment by woqinmian** [comment]
串口部的2-2能输出字符吗，好像输出不了ASCII码吧🥺

**Reply by wintermelon008** [reply]
这里题目表述有点歧义，我们希望串口显示的是字符，而数码管显示的是 ASCII 码。例如，如果开关状态为 8'B0011_0000，那么按下按钮后，串口界面显示 `0`，同时数码管显示 `30`。题目表述会进行对应的更改~
